lib/sles4sap.pm

```diff
@@ -372,7 +372,8 @@ sub prepare_profile {
 
     if ($has_saptune) {
         assert_script_run 'saptune service takeover';
-        my $ret = script_run("saptune solution verify $profile", die_on_timeout => 0);
+        enter_cmd "saptune solution verify $profile; echo DONE-$\? > /dev/$serialdev";
```

Comment:
here is `DONE-$\?` so expanded to `DONE-0` but later the `wait_serial` search for regexp `DONE-\d-` with an extra `-` at the end. Is it fine?

Reply:
true, that's wrong. Fixed. Good catch!